app/javascript/pages/Browse.vue

```diff
@@ -19,6 +19,13 @@
         <BrowserSelector :browser="browser" @clicked="browser = $event" />
       </section>
       <component :is="browser" :contributions="activeContributions" class="row" accessToken="pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"/>
+      <section class="ask-offer">
+        <h2>Can't find what you're looking for?</h2>
+        <div class="actions">
+          <a href="/asks/new" class="button is-primary">Submit an Ask</a>
+          <a href="/offers/new" class="button is-primary">Submit an Offer</a>
+        </div>
+      </section>
```

Comment:
Minor: wherever possible, we've been trying to use existing bulma classes instead of writing custom css.
I think most of the css below can be replaced with these changes.

```suggestion
      <section class="has-text-centered mt-2">
        <h2 class="is-size-4 mb-1">Can't find what you're looking for?</h2>
        <div class="actions">
          <a href="/asks/new" class="button is-primary">Submit an Ask</a>
          <a href="/offers/new" class="button is-primary">Submit an Offer</a>
        </div>
      </section>
```

This doesn't include the horizontal margin between the two buttons. That could be done with `mr-1 ml-1` but these are currently custom classes that we've been meaning to [replace with new bulma spacing classes](https://github.com/rubyforgood/mutual-aid/issues/788), which would make that change cleaner.